tsconfig.json

```diff
@@ -6,7 +6,18 @@
     "jsx": "react-jsx",
     "paths": {
       "@/*": ["./src/*"]
-    }
+    },
+    "types": ["astro/client"],
```

Comment:
No need to add those extra configurations; the tsconfig is extended from `astro/tsconfigs/strict`, so the configuration is already included. 
If you update this to fix a TypeScript issue, updating the include may be sufficient.